Makefile

```
@@ -73,11 +73,16 @@ else
 	go generate
 endif
 
+GIT_DESCRIBE := $(shell git describe --tags 2>/dev/null)
+ifneq ($(GIT_DESCRIBE),)
+BUILD_LDFLAGS = -ldflags "-X actionlint.kjanat.dev.version=$(GIT_DESCRIBE)"
+endif
+
 $(TARGET): $(SRCS)
```

checkmake (0.3.2)

[warning] Target "$(TARGET)" should be declared PHONY.

(phonydeclared)